react-components/src/components/Toolbar/Image360HistoricalOverviewToolbar.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Detail, Flex } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+import { Thumbnail } from '../utils/Thumbnail';
+
+export interface Image360RevisionDetails{
+  revisionDate?: string;
+  revisionImageUrl?: string;
+};
+
+export interface Image360HistoricalOverviewToolbarProps{
+  stationId?: string;
+  stationName?: string;
+  collectionId?: string;
+  revisionCollection?: Image360RevisionDetails[];
+};
```

Comment:
Shouldn't the UI use classes from Reveal that describes the images? And, make revisionDate a `Date`.